src/alias_scraper.py

```diff
@@ -0,0 +1,48 @@
+from urllib.request import urlopen
```

Comment:
Maybe this file belongs under a new `tools` directory. It would also be a good idea to run this as part of the release process: can you add it to the Makefile?